src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -197,25 +225,40 @@ export class EuiSelectableListItem extends Component<
       }
     }
 
+    const instructions = (instruction || state) && (
+      <EuiScreenReaderOnly>
+        <div>
+          {state}
+          {instruction}
+        </div>
+      </EuiScreenReaderOnly>
+    );
+
     return (
       <li
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
-        role="option"
-        aria-selected={!disabled && typeof checked === 'string'}
+        role={role}
+        aria-checked={role === 'option' ? isChecked : undefined}
+        aria-selected={!disabled && isFocused}
```

Comment:
I don't think this is working as expected. I'm looking at https://eui.elastic.co/pr_5581/#/forms/selectable now and every single checked item on the page is showing `aria-selected="false"` even when it is checked. There's not a single `aria-selected="true"` on the page. Should this be 

```suggestion
        aria-checked={role === 'option' ? isChecked : undefined}
        aria-selected={!disabled && isChecked}
```
instead?

Reply:
`aria-selected` is for the faux focused state when an item is highlighted. `aria-checked` is for actual "checked" selection.
This goes back to an earlier review in the `EuiSuggest` PR: https://github.com/elastic/eui/pull/5157#pullrequestreview-780296373

Reply:
Ahhhh. Okay, you're right, and [w3's wai-aria example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/combobox/combobox-autocomplete-none.html) shows the same aria-selected behavior.

It just... feels really weird to me I guess. If I'm tabbing through what's essentially a multi-select, if I hear "selected", I assume it's synonymous with "checked", not "highlighted". Just feels really weird, but who are we to argue with the spec 🤷